README.md

```diff
@@ -6,37 +6,88 @@
+
+_The extension allows to preview a DICOM image and it's most important metadata. On request, all corresponding metadata of the file are displayed in the sidebar._
+
+<img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/2.png" alt="app functionalities" style="width:48%; height:auto;"> &nbsp; &nbsp; <img src="https://github.com/owncloud/awesome-ocis/blob/main/webApps/owncloud/web-app-dicom-viewer/screenshots/3.png" alt="display of metadata" style="width:48%; height:auto;">
+
+_The extension allows to zoom, rotate and flip the preview of the image. Inverting the colors of the preview is also supported._
+
+
+## Adding DICOM Viewer to your oCIS installation
+As oCIS administrator, you can add custom web applications for your users. By adding the DICOM Viewer to the oCIS WebUI, you enable your users to take advantage of the [functionalities of this web extension](#Functionalities-of-the-DICOM-Viewer-Web-Extension).
+
+Have a look at the ownCloud Infinite Scale Deployment documentation to learn how to [extend the WebUI with apps](https://doc.owncloud.com/ocis/next/deployment/webui/webui-customisation.html#extend-web-ui-with-apps). You will find instructions how to [load custom applications](https://doc.owncloud.com/ocis/next/deployment/webui/webui-customisation.html#loading-applications) into your installation and get a better understanding of the web extension [application structure](https://doc.owncloud.com/ocis/next/deployment/webui/webui-customisation.html#application-structure) and [application configuration](https://doc.owncloud.com/ocis/next/deployment/webui/webui-customisation.html#application-configuration).
+
+---
+
+DICOM Viewer's `manifest.json` (referenced in `application structure`) can be found [here](public/manifest.json).
+
+TODO:
+- find out if `apps.yaml` file is needed (seems to be optional)
+- if so, provide corresponding file and link to it?
+- see draft PR https://github.com/owncloud/web-app-dicom-viewer/pull/67
+
+---
+
+### Prerequisites
+
+#### Supported oCIS and Web Versions
+- oCIS (>= 6.2.x)
+- Web (>= 9.x.x)
+
+#### Supported architectures:
+    `amd64`
+
+### App Installation
+
+1. Download the zip file from the [releases page](https://github.com/owncloud/web-app-dicom-viewer/releases)  
+2. Extract the zip file to the `apps` directory of the oCIS server. The `apps` directory is set using the `WEB_ASSET_APPS_PATH` environment variable.
+
+
+## Build and run DICOM Viewer for development
+
+### Prerequisites
+
+- [Node.js `v18`](https://nodejs.org/en/)
+- [pnpm `v8`](https://pnpm.io/)
+- [Docker Compose](https://docs.docker.com/compose/)
```

Comment:
@Salipa-Gurung this is taken from presentation viewer documentation, can you please check if this is correct?

Reply:
this works for dicom viewer app also.

Reply:
thanks for confirming!